test/item_test.rb

```diff
@@ -9,7 +9,12 @@ def setup
     @item_name = "DesignerEstore"
     @item_description = "Disney glitter frames"
     @item_merchant_id = 12334141
-    @item = Item.new({:id => @item_id, :name => @item_name, :description => @item_description, :merchant_id => @item_merchant_id})
```

Comment:
Nice! @alfosco - this looks much clearer with the new item broken out onto several lines.  Good call.